resources/quality/dagoma_pro_430_bowden/Steel nozzle 0.4/dagoma_pro_430_bowden_0.4_abs_high_quality.inst.cfg

```diff
@@ -0,0 +1,58 @@
+[general]
+
+version = 4
+name = High Quality
+definition = dagoma_pro_430_bowden
+
+[metadata]
+
+setting_version = 19
+type = quality
+quality_type = high_quality
+weight = -1
+material = dagoma_generic_abs
+variant = 0.4
+
+[values]
+
+wall_thickness = =machine_nozzle_size * 2
+optimize_wall_printing_order = True
+skin_no_small_gaps_heuristic = True
+fill_outline_gaps = True
```

Comment:
This value is always true. So it shouldn't be set in the quality profile but in the definition. 

Note that there are more settings like this. Take care to set as many values in a profiles as deep down the stack as possible!